samples/drivers/i2s/echo/boards/nrf52840dk_nrf52840.overlay

```diff
@@ -4,10 +4,37 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+&pinctrl {
+	i2c0_default_alt: i2c0_default_alt {
+		group1 {
+			psels = <NRF_PSEL(TWIM_SDA, 0, 30)>,
+				<NRF_PSEL(TWIM_SCL, 0, 31)>;
+		};
+	};
+
+	i2c0_sleep_alt: i2c0_sleep_alt {
+		group1 {
+			psels = <NRF_PSEL(TWIM_SDA, 0, 30)>,
+				<NRF_PSEL(TWIM_SCL, 0, 31)>;
+			low-power-enable;
+		};
+	};
+
+	i2s0_default_alt: i2s0_default_alt {
+		group1 {
+			psels = <NRF_PSEL(I2S_SCK_M, 0, 15)>,
+				<NRF_PSEL(I2S_LRCK_M, 0, 12)>,
+				<NRF_PSEL(I2S_SDOUT, 0, 13)>,
+				<NRF_PSEL(I2S_SDIN, 0, 14)>;
```

Comment:
Incorrect translation of pin numbers (same story in nrf5340dk_nrf5340_cpuapp.overlay). It should be:
```suggestion
			psels = <NRF_PSEL(I2S_SCK_M, 1, 15)>,
				<NRF_PSEL(I2S_LRCK_M, 1, 12)>,
				<NRF_PSEL(I2S_SDOUT, 1, 13)>,
				<NRF_PSEL(I2S_SDIN, 1, 14)>;
```

Reply:
fixed, thanks